### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by Kelketek**
Updates to the documentation for this XBlock.

This may be most easily viewed by just going to the [`README.md`](https://github.com/open-craft/xblock-poll/blob/doc-updates/README.md).

Also included is a small fix that solves an issue where a course author couldn't submit a vote right after adding a poll to their unit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,178 @@ and images. Formatting for images is handled by the XBlock's formatters to keep
...
+![Poll example results](doc_img/poll_result.png)
+
+The top choice's percentage is shown in *orange* while the user's selection is marked by a selected (but disabled)
+radio button on the side.
+
+### Poll variations
+
+Polls may have several customizations. Feedback may be added to a poll for display after the user has submitted their
+answer. Answers may have images associated with them. The question, feedback, and answers are all permitted to contain
+markdown (as well as arbitrary HTML) and so may be customized in the studio (as discussed in the next chapter).
```
Comment (antoviaque) [inline README.md]: Could be useful to add a link to the markdown specification

File: README.md [hunk 2]
```diff
@@ -19,12 +19,178 @@ and images. Formatting for images is handled by the XBlock's formatters to keep
...
+
+![Image-only poll](doc_img/img_poll.png)
+
+This poll also contains a feedback section, which is enhanced with Markdown:
+
+![Image-only poll results](doc_img/img_poll_result.png)
+
+Polls may also have images and with text.
+
+![Image and Label label poll](doc_img/img_and_label_poll.png)
```
Comment (antoviaque) [inline README.md:79]: Very good choice of poll images ;p

File: README.md [hunk 3]
```diff
@@ -19,12 +19,178 @@ and images. Formatting for images is handled by the XBlock's formatters to keep
...
+* Answers
+
+The following fields are customizable with Markdown and custom HTML on Surveys:
+
+* Feedback
+* Questions
+
+You may add another answer (or question, if creating a survey) by clicking the add links at the bottom of the form:
+
+![Add links](doc_img/add_links)
```
Comment (antoviaque) [inline README.md]: Wrong URL (the image doesn't show)

**Comment by antoviaque** [comment]
Besides minors nits, :+1:

**Comment by Kelketek** [comment]
@antoviaque Your comments have been addressed. :)

**Comment by antoviaque** [comment]
@Kelketek Thanks!

**Pull request merged**